README.md

```diff
@@ -0,0 +1,6 @@
+## DeskNotes
+DeskNotes is a tool to put simple sticky notes on the Desktop 
```

Comment:
Full stop at the end.

Reply:
I have updated the screenshot to show this feature.